dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AliasableResourceInterface.java

```diff
@@ -68,11 +65,7 @@ default T updateAliases(User user, Long id, String aliases, String emptyBody) {
             }
         }
 
-        Set<String> aliasesToAdd = Sets.difference(newAliases, oldAliases);
-        Set<String> aliasesToRemove = new TreeSet<>(Sets.difference(oldAliases, newAliases));
-        // add new ones and remove old ones while retaining the old entries and their order
-        aliasesToAdd.forEach(alias -> c.getAliases().put(alias, new Alias()));
-        aliasesToRemove.forEach(alias -> c.getAliases().remove(alias));
+        newAliases.forEach(alias -> c.getAliases().put(alias, new Alias()));
```

Comment:
What happens if somebody adds the same alias? Before it would have been ignored, now you will try to add it again, I think. If you detect somebody trying to add an existing alias, you can either silently fail or return a bad request. I lean towards bad request.

Reply:
+1 for bad request

Reply:
done

Reply:
done, now issues bad request

Reply:
done, updated tests